src/interfaces/coral_web/src/utils/getSlugRoutes.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * @description This funciton is used to parse the slug from the URL and return the agentId and conversationId.
+ * The slug can be in the following formats:
+ * - [] - /
+ * - [c, :conversationId] - /c/:conversationId
+ * - [:agentId] - /:agentId
+ * - [:agentId, c, :conversationId] - /:agentId/c/:conversationId
+ */
+
+const isUUID = (value: string) => {
+  const isValidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
+
+  return isValidRegex.test(value);
+};
+
+export const getSlugRoutes = (
+  slugParams?: string | string[]
+): {
+  agentId: string | undefined;
+  conversationId: string | undefined;
+} => {
+  if (!slugParams || typeof slugParams === 'string')
+    return { agentId: undefined, conversationId: undefined };
+
+  // firstQuery = :agentId | c | undefined
```

Comment:
Just adding for clarity
```suggestion
  // Possible values are:
  // firstQuery = :agentId | c | undefined
```